### PR TITLE
add missing `test2` index in alias example

### PR DESCRIPTION
**Pull request by talevy**
from original PR:

>If I got the idea of aliases properly, I think that the index "test2" should have a reference in the 
>example above of the following sentence:
>
> " ... we associate the alias `alias1` to both `test` and `test2` ... "

this PR re-applies #39055. That PR was prematurely merged with a failing test.

### CODE REVIEW

File: docs/reference/indices/aliases.asciidoc [hunk 1]
```diff
@@ -269,12 +269,18 @@ POST /_aliases
...
+            }
         }
     ]
 }
 --------------------------------------------------
 // CONSOLE
-// TEST[s/^/PUT test\n/]
+// TEST[s/^/PUT test\nPUT test2\n/]
 
 In this example, we associate the alias `alias1` to both `test` and `test2`, where
```
Comment (jtibshirani) [inline docs/reference/indices/aliases.asciidoc]: Reading the rest of this example, I think the final section may be incorrect as well:

```
To swap which index is the write index for an alias, the Aliases API can be leveraged to
do an atomic swap. The swap is not dependent on the ordering of the actions.

POST /_aliases
{
    "actions" : [
        {
            "add" : {
                 "index" : "test",
                 "alias" : "alias1",
                 "is_write_index" : true
            }
        }, {
            "add" : {
                 "index" : "test2",
                 "alias" : "alias1",
                 "is_write_index" : false
            }
        }
    ]
}
```

Given the description, it seems like the boolean values for `is_write_index` should be flipped. I think it'd be nice to fix that in this PR as well, if you also think that's reasonable?

Comment (talevy) [reply docs/reference/indices/aliases.asciidoc]: ah, I see. I think, when righting the docs, this was meant to be thought of as a separate example. Now I see that it feels like a continuation of the previous

**Comment by talevy** [comment]
thanks Julie, updated!

**Review by jtibshirani — APPROVED** [review]
Looks good to me!

**Comment by talevy** [comment]
thanks!

**Pull request merged**